src/app/auth/login/login.component.spec.ts

```diff
@@ -0,0 +1,114 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { FormBuilder, ReactiveFormsModule } from '@angular/forms';
+import { UsuarioService } from 'src/app/services/usuario.service';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ComfyStream/ComfyStream-Frontend/pullRequest?prid=7127387)
```suggestion
import { HttpClientTestingModule } from "@angular/common/http/testing";
```